beelzebub-chart/templates/tests/test-connection.yaml

```
@@ -8,8 +8,10 @@ metadata:
     "helm.sh/hook": test
 spec:
   containers:
+    {{- range $value := .Values.beelzebubServiceConfigs }}
     - name: wget
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)